libs/platform/src/lib/components/wizard-generator/components/dialog-wizard-generator/dialog-wizard-generator.component.ts

```diff
@@ -0,0 +1,94 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, TemplateRef, ViewChild, ViewEncapsulation } from '@angular/core';
+import { DialogRef, DialogService } from '@fundamental-ngx/core/dialog';
+import { filter, takeWhile } from 'rxjs/operators';
+import { BaseWizardGenerator } from '../../base-wizard-generator';
+import { WizardDialogData } from '../../interfaces/wizard-dialog-data.interface';
+import { WizardTitle } from '../../interfaces/wizard-title.interface';
+import { WizardGeneratorService } from '../../wizard-generator.service';
+
+@Component({
+    selector: 'fdp-dialog-wizard-generator',
+    templateUrl: './dialog-wizard-generator.component.html',
+    styleUrls: ['./dialog-wizard-generator.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [WizardGeneratorService]
+})
+export class DialogWizardGeneratorComponent extends BaseWizardGenerator {
+
+    @ViewChild('confirmationDialog') confirmationDialog: TemplateRef<any>;
+
+    /**
+     * @description Wizards dialog title configuration.
+     */
+    title: WizardTitle;
+
+    confirmationDialogText = 'Are you sure you want to discard your progress?';
```

Comment:
What about i18n for these strings, will it be working?

Reply:
Users can pass their own translated text here with Input for component, or inside config object for dialog

Reply:
since we can use templateRef as input options we probably can leave it as it is.